### PR TITLE
Savannah Layout & added

### PR DESCRIPTION
**Pull request by Ars6300**
Savannah Layout & added shared button for fullscreen games
image

### CODE REVIEW

File: src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="savannah">
+  <div class="savannah-words">
+    <button mat-button class="savannah-words_item" *ngFor="let word of words; let i = index">
```
Comment (rss-maksim) [inline src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.html:3]: trackBy index?

File: src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+:host {
+  position: relative;
+  background: url("../../shared/assets/savannah-background.jpg") no-repeat center center fixed;
+  // filter: blur(3px);
```
Comment (rss-maksim) [inline src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.scss:4]: Is it necessary at the moment?

File: src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+:host {
```
Comment (rss-maksim) [inline src/app/mini-games/savannah/savannah-mini-game/savannah-mini-game.component.scss:1]: BEM?

File: src/app/mini-games/shared/components/full-screen-button/full-screen-button.component.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+@Component({
+  selector: 'app-full-screen-button',
+  templateUrl: './full-screen-button.component.html',
+  styleUrls: ['./full-screen-button.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FullScreenButtonComponent implements OnInit {
+  @Input() color = 'primary'; // primary || accent || warn || disabled
+  elem: any;
```
Comment (rss-maksim) [inline src/app/mini-games/shared/components/full-screen-button/full-screen-button.component.ts:12]: No so good to use ```any```

File: src/app/mini-games/shared/components/full-screen-button/full-screen-button.component.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  @HostListener('document:webkitfullscreenchange', ['$event'])
+  @HostListener('document:mozfullscreenchange', ['$event'])
+  @HostListener('document:MSFullscreenChange', ['$event'])
+  fullscreenmodes(_event: any) {
+    this.checkScreenMode();
+  }
+
+  checkScreenMode() {
+    if (document.fullscreenElement) {
+      this.isFullScreen = true;
```
Comment (rss-maksim) [inline src/app/mini-games/shared/components/full-screen-button/full-screen-button.component.ts:31]: @suggestion
```this.isFullScreen = Boolean(document.fullscreenElement)```

**Review by rss-maksim — APPROVED** [review]

**Review by IlyaKozak — APPROVED** [review]

**Pull request merged**